components/common/src/command/package/install.rs

```diff
@@ -72,13 +73,18 @@ where
     P2: AsRef<Path> + ?Sized,
 {
     if !am_i_root() {
-        ui.warn(
-            "Installing a package requires root or administrator privileges. Please retry \
+        if let Some(_) = env::var_os("HAB_NON_ROOT") {
+            ui.warn("Allowing install by non-root user!")?;
```

Comment:
It might be useful to be explicit in the warning message that this is because `HAB_NON_ROOT` was set; that way it is absolutely clear from the message what is happening and _why_ it is happening.

Reply:
@christophermaier I think we can just remove the warning actually. If you set this env var you know full well what you're doing and it's documented in the docs above ;)

Reply:
Ah, I had a bit of a brain-fart here... this is UI messaging, not logging. In that case, then yeah, I'm with you 100% @reset

Reply:
I removed this in a later commit

Reply:
@bodymindarts I like the new code... very concise 👍